components/authentication/base/rhtap-admins.yaml

```diff
@@ -19,6 +19,7 @@ rules:
       - releaseplanadmissions
       - releaseplans
       - releases
+      - releaseserviceconfigs
```

Comment:
Could be done in another PR or this one but we should add `releaseserviceconfigs` resource to the `view-appstudio` ClusterRole in "infra-deployments/components/authentication/base/everyone-can-view.yaml".

We grant view access to any RHTAP resources to all RHTAP developers in that manifest.

Reply:
added.